src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {Icon} from '../../../Icon/Icon';
+import {MediaRenderer} from '../../../Stories/components';
+import {block} from '../../../utils/cn';
+import {StoriesGroupItem} from '../../types';
+import {Chevron} from '../../../icons/Chevron';
+import './StoriesPreview.scss';
+import {Button} from '../../../Button';
+
+const b = block('stories-group-preview');
+
+type StoriesPreviewListProps = {
+    groups: StoriesGroupItem[];
+    groupIndex: number;
+    onGroupSelect: (groupIndex: number) => void;
+    offset?: number;
```

Comment:
Should it be customized? I cannot imagine scenario when we need to use custom offset

Reply:
https://github.com/gravity-ui/uikit/blob/ca146d5d6acc826c00f08185a2340b48a4a4c5b2/src/components/StoriesGroup/components/StoriesPreview/StoriesPreview.tsx#L90